appimage/run_appimage.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+PYTHON_CMD="$APPDIR"/opt/python3.7/bin/python3.7
+DOT_MONKEY="$HOME"/.monkey_island/
+
+configure_default_logging() {
+    if [ ! -f "$DOT_MONKEY"/island_logger_config.json ]; then
+        cp "$APPDIR"/usr/src/island_logger_config.json "$DOT_MONKEY"
+    fi
+}
+
+configure_default_server() {
+    if [ ! -f "$DOT_MONKEY"/server_config.json ]; then
+        cp "$APPDIR"/usr/src/monkey_island/cc/server_config.json.standard "$DOT_MONKEY"/server_config.json
+    fi
+}
+
+
+
+
+# shellcheck disable=SC2174
+mkdir --mode=0700 --parents "$DOT_MONKEY"
+
+DB_DIR="$DOT_MONKEY"/db
+mkdir --parents "$DB_DIR"
+
+configure_default_logging
+configure_default_server
```

Comment:
Wait, so this file runs once on setup, right? Because if it runs each time you wanna run monkey island, this will reset the account each time.

Reply:
Correct.
```bash
configure_default_server() {
    if [ ! -f "$DOT_MONKEY"/server_config.json ]; then
        cp "$APPDIR"/usr/src/monkey_island/cc/server_config.json.standard "$DOT_MONKEY"/server_config.json
    fi
}
```

Reply:
Documentatino

Reply:
We need a broader task to update the documentation when we decide if we're abandoning the OVA images, deb package, etc. Maybe this should wait until then.